src/components/QueueApp/QueueApp.module.css

```diff
@@ -1,13 +1,28 @@
 .QueueApp {
+  font-family: Raleway;
```

Comment:
Does the font not import from the index.css? Just wondering.

Reply:
Great question. I noticed that Raleway was not rendering as expected, so this was my band-aid approach to fixing the problem.

I took a closer look and found out what the problem was. Tailwind is overriding Raleway as the default font-style. The proper approach is to change the tailwind default

https://tailwindcss.com/docs/font-family

Which I did here (in #89)

https://github.com/0x213F/jukebox-radio-react/pull/89/commits/4883e2178f9d5533442a8a5940893af2ff8a76a3

Reply:
Oh I see! Makes more sense.